bitnami/kube-prometheus/templates/exporters/kube-controller-manager/service.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.kubeControllerManager.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "kube-prometheus.fullname" . }}-kube-controller-manager
+  namespace: kube-system
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app: {{ template "kube-prometheus.fullname" . }}-kube-controller-manager
```

Comment:
Same for the selector

Reply:
please check the below comment regarding selector